src/unix/process.c

```diff
@@ -359,6 +357,8 @@ int uv_spawn(uv_loop_t* loop,
   int exec_errorno;
   int i;
   int status;
+  sigset_t sigiset;
+  sigset_t sigset;
```

Comment:
Consider renaming this. `sigset` is an obsolete-ish alternative to `sigaction()` but I think POSIX allows it to be implemented as a macro (which would expand to nonsense here.)